index.html

```diff
@@ -40,6 +45,164 @@ <h2 class="headline-subHeading">I’m a Software Developer</h2>
+            </ul>
+            <button class="project-Button">See Project</button>
+        </article>
+
+        <article class="card project5">
+            <header class="project-Header">
+                <h1>Profesional Art Printing Data</h1>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required.
+                    has been the industry's standard</p>
+            </header>
+            <ul>
+                <li><a href="https://en.wikipedia.org/wiki/HTML" class="program-Link">html</a></li>
+                <li><a href="https://en.wikipedia.org/wiki/Bootstrap_(front-end_framework)" class="program-Link">bootstrap</a></li>
+                <li><a href="https://en.wikipedia.org/wiki/Ruby" class="program-Link">Ruby</a></li>
+            </ul>
+            <button class="project-Button">See Project</button>
+        </article>
+
+        <article class="card project6">
+            <header class="project-Header">
+                <h1>Profesional Art Printing Data</h1>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required.
+                    has been the industry's standard</p>
+            </header>
+            <ul>
+                <li><a href="https://en.wikipedia.org/wiki/HTML" class="program-Link">html</a></li>
+                <li><a href="https://en.wikipedia.org/wiki/Bootstrap_(front-end_framework)" class="program-Link">bootstrap</a></li>
+                <li><a href="https://en.wikipedia.org/wiki/Ruby" class="program-Link">Ruby</a></li>
+            </ul>
+            <button class="project-Button">See Project</button>
+        </article>
+
+        <article class="card project7">
+            <header class="project-Header">
+                <h1>Profesional Art Printing Data</h1>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required.
+                    has been the industry's standard</p>
+            </header>
+            <ul>
+                <li><a href="https://en.wikipedia.org/wiki/HTML" class="program-Link">html</a></li>
+                <li><a href="https://en.wikipedia.org/wiki/Bootstrap_(front-end_framework)" class="program-Link">bootstrap</a></li>
+                <li><a href="https://en.wikipedia.org/wiki/Ruby" class="program-Link">Ruby</a></li>
+            </ul>
+            <button class="project-Button">See Project</button>
+        </article>
+    </section>
+
+    <section id="AboutMe">
+        <header class="aboutme-Header">
+            <h1>About me</h1>
+            <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of
+                my work and experience! If you like what you see and have a project you need coded, don’t hestiate to
+                contact me.</p>
+            <button class="get-Resume-Button">Get My resume</button>
+        </header>
```

Comment:
- [ ] Please replace the `<header>` tag with a `div`

Reply:
Thank You for your guidance. I've made all the required changes :grin: